charts/actions-runner-controller/templates/webhook_configs.yaml

```
@@ -219,3 +237,18 @@ webhooks:
     resources:
     - runnerreplicasets
   sideEffects: None
+{{ if not (and .Values.admissionWebHooks.caBundle .Values.certManagerEnabled) }}
```

```suggestion
{{ if not (or .Values.admissionWebHooks.caBundle .Values.certManagerEnabled) }}
```

If I'm not missing anything, shouldn't this be `or` rather than `and`? With `and`, it can result in using Helm-generated key and cert even though you set `certManagerEnabled=true`, or `admissionWEbhooks.caBundle=whatever,certManagerEnabled=false`.

---

Indeed, my boolean logic is a bit rusty